src/components/EmptyArea.tsx

```diff
@@ -0,0 +1,45 @@
+import { Box, Center, VStack, Text, HTMLChakraProps } from "@chakra-ui/react"
```

Comment:
same comments as on prior pr!

Reply:
Hmm, gotcha, we shall address this is that other then!